README.md

```diff
@@ -19,13 +19,17 @@ module "state" {
 <!-- BEGIN_TF_DOCS -->
 ## Requirements
 
-No requirements.
+The following requirements are needed by this module:
+
+- terraform (>=1.0.0)
```

Comment:
Since the [release notes states to use the current terraform version (https://learn.microsoft.com/en-us/azure/developer/terraform/provider-version-history-azurerm-4-0-0-to-current#400-august-22-2024) I'd suggest to at least put a hint about this here.

> When upgrading to v4.0 of the AzureRM Provider, we recommend upgrading to the latest version of Terraform Core ([which can be found here](https://www.terraform.io/downloads)).

Reply:
This is generated documentation from the terraform.tf, but I'll update the terraform requirement to 1.9 maybe as well then